flight/Modules/Sensors/simulated/simsensors.c

```diff
@@ -85,7 +86,9 @@ static float rand_gauss();
 
 static bool use_real_sensors;
 
-enum sensor_sim_type {CONSTANT, MODEL_AGNOSTIC, MODEL_QUADCOPTER, MODEL_AIRPLANE, MODEL_CAR} sensor_sim_type;
+extern bool use_yasim;
```

Comment:
Would be nice to pass this through nicely somehow. PIOS_Sensors seems like the right place.. but not specific to sim currently :/.